assets/src/scripts/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import SearchBarResults from './SearchBarResults.js';
+
+import 'whatwg-fetch';
+
+var strings = {
+	label:       'Search',
+	button:      'Submit',
+	placeholder: 'Search the site…',
+};
+
+export default class SearchBar extends React.Component {
```

Comment:
just a small personal preference, I like to `import React, { Component }` then just `extends Component`